src/main/resources/config.properties

```diff
@@ -13,6 +13,7 @@ builder.matlab.automatictestoption.display.name = Automatic
 builder.matlab.customcommandoption.display.name = Custom
 Releaseinfo.matlab.version.not.found.error = Error finding MATLAB release for given MATLAB root. Verify MATLAB root path.
```

Comment:
This error message needs revision. How different is it from the message below it?

Reply:
This message is relevant for both 'MATLAB root' box in 'Use MATLAB Version' and the 'MATLAB root' box in 'Global Tool Configuration' page. This error message wouldn't need revision.

It differs from 'Unable to launch MATLAB... ' error message as this error only implies it's not able to find MATLAB release version and that the plugin wouldn't know if plugin supports a particular version of MATLAB. This is an unlikely edge case as MATLAB installation directory should have a Contents.m or VersionInfo.xml file.

Reply:
Thanks for clarifying. I just think the error message needs to be reworded. It sounds awkward. How about something like this?

Unable to find the MATLAB version corresponding to the specified path. Verify the path to MATLAB root folder.

* find or locate